src/github/client/github-client.types.ts

```diff
@@ -1,6 +1,6 @@
 import { AxiosResponse } from "axios";
-import { Octokit } from "probot";
 import { GraphQLError } from "./github-client-errors";
+import { Octokit } from "probot";
```

Comment:
We're trying to remove this. Maybe just copy the pertinent interface over instead?

Reply:
I didn't add this (my formatter just changed the order of the imports)